tests/php/test-class-amp-theme-support.php

```diff
@@ -1043,17 +1044,27 @@ function() {
 		AMP_Theme_Support::init_admin_bar();
 		$this->assertEquals( 10, has_filter( 'style_loader_tag', [ AMP_Theme_Support::class, 'filter_admin_bar_style_loader_tag' ] ) );
 		$this->assertEquals( 10, has_filter( 'script_loader_tag', [ AMP_Theme_Support::class, 'filter_admin_bar_script_loader_tag' ] ) );
-		$this->assertFalse( has_action( 'wp_head', $callback ) );
+
+		if ( ! function_exists( 'wp_enqueue_admin_bar_header_styles' ) ) {
+			$this->assertFalse( has_action( 'wp_head', $callback ) );
+		}
+
+		remove_action( 'wp_print_styles', 'print_emoji_styles' );
```

Comment:
Could you add a `//` comment after this and the other instances in this file just to explain why it is removed for future reference?

Reply:
Done in https://github.com/ampproject/amp-wp/pull/7635/commits/82fe0888b09a1de55d204b27a544c4ae1628a0c9.

Reply:
In `default-filters.php` in core now, it has:

```php
add_action( 'admin_print_styles', 'print_emoji_styles' ); // Retained for backwards-compatibility. Unhooked by wp_enqueue_emoji_styles().
```

So isn't this handled by `wp_enqueue_emoji_styles()`?

Reply:
`is_admin()` is false at this point.

https://github.com/ampproject/amp-wp/blob/3cd9b29fa7226af3b2184f579cff5e3095e2f790/tests/php/test-class-amp-theme-support.php#L55